templates/media/media--image.html.twig

```diff
@@ -0,0 +1,49 @@
+{#
+/**
+ * @file
+ * Theme override to display an image media item with WA-compliant description.
+ * ~ When field_aria_description has value, this template:
+ * ~~ Adds aria-describedby="description-[MID]" to field_media_image <img> tag.
+ * ~~ Renders field_aria_description inside div#description-[MID].sr-only.
+ * See cwd_base/media/media.html.twig for available variables and other info.
+ *
+ * @see template_preprocess_media()
+ *
+ * @ingroup themeable
+ */
+#}
+{{ title_suffix.contextual_links }}
+{% if content %}
+
+  {% if media.field_aria_description.0.value -%}
+    {% if media.field_media_image.0.value -%}
```

Comment:
Personal preference, I include lots of value checks, b/c the downside of "unanticipated situation" is a WSOD.  (would be cool if missing values didn't cause WSOD... kinda feels like that should be one of the benefits of using something snazzy like twig instead of straight PHP... but :woman_shrugging: )